library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -752,7 +761,7 @@ def Read
       @Devices = {}
 
       # preparation
-      devices = get_devices
+      devices = get_devices(ignore_confs_regex)
```

Comment:
Apparently this is the only use of that method, which now has a default argument of `"[~]"`. Let's update it to accept a Regexp and remove the default for the argument.
Return `Array<String>`, not `Array`.
Make it private?